test/test-conditions-3.js

```diff
@@ -0,0 +1,5 @@
+const operations = require("./operations-conditions-3.js");
+const assert = require("assert");
+it("Правильно определяет оценки conditions-task-3", () => {
+  assert.equal(operations.assignGrade(62, 75), "mark is 4");
```

Comment:
you need more test cases here

Reply:
Why you are passing 2 arguments here?
```
operations.assignGrade(62, 75)
```

When your function expects only one and will ignore additional